getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,73 @@
+# Instructions for macOS
+
+Install [Homebrew](http://brew.sh/) to give your mac a package manager. Open up your terminal emulator of choice and run:
+
+```shell
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Install git:
+
+```shell
+$ brew install git
```

Comment:
Pretty sure that's only if you install the XCode dev tools, which I avoided installing in this guide.